src/mono/monoaotcross.proj

```diff
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="BuildMonoCrossAllTargets">
```

Comment:
Same comment about the default target. Just remove the attribute here and use an aftertargets="Build" in the `BuildMonoCrossAllTargets` target.